### PR TITLE
upgraded to next prod version

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

https://github.com/kiegroup/kie-jenkins-scripts/pull/1225
https://github.com/kiegroup/kie-jenkins-scripts/pull/1226

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy [hunk 1]
```diff
@@ -13,8 +13,8 @@ def NEXT_PRODUCT_BRANCH='main'
 def NEXT_RHBA_VERSION_PREFIX=Constants.RHBA_VERSION_PREFIX
 
 def CURRENT_PRODUCT_VERSION=Constants.CURRENT_PROD_VERSION
```
Comment (Ginxo) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:15]: is https://github.com/kiegroup/kie-jenkins-scripts/blob/ba40179a95eba368e3c0666c919893b2ce90ceef/job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy#L41 value ok? shouldn't it be `7.13.0`? and what about `NEXT_PROD_VERSION`? :thinking:

**Review by Ginxo — CHANGES_REQUESTED** [review]
please check

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy (hunk 1 above)
Comment (mbiarnes) [reply job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:15]: https://github.com/kiegroup/kie-jenkins-scripts/pull/1226/files

**Comment by mbiarnes** [comment]
due to the "end" of the 7.x stream versions (latest will be 7.13 - like already defined) it doesn't make sense to upgrade the Constants - 7.14 will never exist. Right now we agreed to have 7.12 as CURRENT and 7.13 as  NEXT version.

**Pull request closed without merging**